auth/Makefile

```diff
@@ -0,0 +1,90 @@
+DIR=certs
+PROTOCOL=x509
+KEY_SIZE=4096
+VALIDITY_DAYS=9999
+
+CA_KEY=$(DIR)/ca-key.pem
+CA_CRT=$(DIR)/ca-crt.pem
+
+VMD_SERVER_KEY=$(DIR)/vmd-server-key.pem
```

Comment:
Please rename all these to something like `-sample-` to indicate that they are only for testing. Whoever uses these must be super clear that's the case. Both for server and client.